src/components/NodeToolbar/Photos/CtaPhotoUpload.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const CtaPhotoUpload = props => (
```

Comment:
I guess this means 'call to action', but maybe this could have a simpler name?

Reply:
Is unused atm, I can probably delete it.